server/services/team/service.ts

```diff
@@ -566,13 +568,9 @@ export class TeamService extends ServiceClass implements ITeamService {
 	}
 
 	async removeMembers(teamId: string, teamName: string, members: Array<ITeamMemberParams>): Promise<void> {
-		if (!teamId) {
-			const teamIdName = await this.TeamModel.findOneByName(teamName, { projection: { _id: 1 } });
-			if (!teamIdName) {
-				throw new Error('team-does-not-exist');
-			}
-
-			teamId = teamIdName._id;
+		const team = await this.TeamModel.findOneByIdOrName(teamId || teamName, { projection: { _id: 1, roomId: 1 } });
```

Comment:
This isn't safe. It's better to search for id when we get an id and search for name when we get a name.

Reply:
Just curious, why do you say it's not safe? 👀

Reply:
there can be a room with the name of another room's id (unlikely, but possible). Then the query could pick the wrong room here.